mock-backend/src/data/mokedOrders.js

```diff
@@ -143,6 +143,7 @@ const adminOrders = {
       id: "126bf189-9f8c-46ed-9138-f74d9d93206e",
       isPaid: true,
       orderStatus: "IN_PROGRESS",
+      availableStatuses: ["SHIPPED", "DELIVERED", "COMPLETED", "CANCELED"],
```

Comment:
Same

Reply:
Its backend! We don't need this constant there